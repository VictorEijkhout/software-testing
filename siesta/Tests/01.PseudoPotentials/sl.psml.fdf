SystemLabel         sl.psml

%block ChemicalSpeciesLabel
 1  14  Si.sl  Si.tm2.nrl.vnl.psml
%endblock ChemicalSpeciesLabel

%include ../base_si2.fdf