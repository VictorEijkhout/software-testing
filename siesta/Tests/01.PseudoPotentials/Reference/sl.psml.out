Siesta Version  : 5.0.0-beta1-29-g209a221d1
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
NetCDF support
NetCDF-4 support
Lua support
DFT-D3 support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/01.PseudoPotentials/work/sl.psml
* Running in serial mode.
>> Start of run:  14-DEC-2023  13:38:20

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.81976
************************** Dump of input data file ****************************
SystemLabel         sl.psml
%block ChemicalSpeciesLabel
 1  14  Si.sl  Si.tm2.nrl.vnl.psml
%endblock ChemicalSpeciesLabel
%include ../../base_si2.fdf
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: Si bulk
reinit: -----------------------------------------------------------------------
reinit: System Label: sl.psml
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   14 Label: Si.sl


---- Processing specs for species: Si.sl
Ground state valence configuration:   3s02  3p02
Reading pseudopotential information in PSML from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/Si.tm2.nrl.vnl.psml
PSML file version: 1.1
Using libxc ids:    1   9
LDA--CA XC_LDA_X--XC_LDA_C_PZ ca
PSML uuid: 892ccc00-7303-11e7-6e21-1ea5ed9f1753

---- Pseudopotential check for Si

Pseudized shells:
3s( 2.00) rc: 1.89
3p( 2.00) rc: 1.89
3d( 0.00) rc: 1.89
4f( 0.00) rc: 1.89
Valence configuration for ps generation: 3s:3p: Total charge:   4.000000
For Si.sl, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
Si.sl                Z=  14    Mass=  28.090        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=3
          i=1  nzeta=1  polorb=0  (3s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=3
          i=1  nzeta=1  polorb=1  (3p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=0  polorb=0  (3d)  (perturbative polarization orbital) (from 3p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for Si.sl                 (Z =  14)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM4.2.6  Troullier-Martins                       
Valence charge in pseudo generation:    4.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
Got Vlocal (siesta-fit) from psml data
Got Chlocal from  psml data
 rchloc from  annotation, nchloc:    2.8530272231199998             5658
atom: Maximum radius for chlocal:    2.85288
atom: Maximum radius for r*vlocal+2*Zval:    2.82157
Reading KB projs from Si.sl psml data
Using non-relativistic projectors in PSML file

PSML: Kleinman-Bylander projectors: 
   l= 0   rc=  1.922530   Ekb=  4.661340
   l= 1   rc=  1.922530   Ekb=  1.494238
   l= 2   rc=  1.920598   Ekb= -2.808672
   l= 3   rc=  1.955663   Ekb= -0.959059

PSML: Total number of  Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 3s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    5.413980
                 energy =   -0.785062
                kinetic =    0.554059
    potential(screened) =   -1.339121
       potential(ionic) =   -3.805086

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 3p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    6.830288
                 energy =   -0.295476
                kinetic =    0.850831
    potential(screened) =   -1.146306
       potential(ionic) =   -3.389842

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 3p
Split based on tail norm

   izeta = 1
                     rc =    6.830288
                 energy =    0.328538
                kinetic =    1.535909
    potential(screened) =   -1.207371
       potential(ionic) =   -3.334649
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 3s( 2.00)                                                            
 3p( 2.00)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    4.00000   4.00000

Vna:  Cut-off radius for the neutral-atom potential:   6.830288

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   14 Si.sl                   # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
Si.sl                 2                    # Species label, number of l-shells
 n=3   0   1                         # n, l, Nzeta 
   5.414   
   1.000   
 n=3   1   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   6.830   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file Si.sl.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in units of alat)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      2.56530   2.56530   2.56530  1        2

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      2    18    32

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   150.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   T
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9863 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:     8.000000
Total ionic charge:     8.000000
 Kpoints in:           18 . Kpoints trimmed:           14

siesta: k-grid: Number of k-points = 14
siesta: k-points from Monkhorst-Pack grid
siesta: k-cutoff (effective) =     5.759 Ang
siesta: k-point supercell and displacements
siesta: k-grid:     3     0     0      0.000
siesta: k-grid:     0     3     0      0.000
siesta: k-grid:     0     0     3      0.000

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000

superc: Internal auxiliary supercell:     5 x     5 x     5  =     125
superc: Number of atoms, orbitals, and projectors:    250   2250   4000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     5 x     5 x     5  =     125
superc: Number of atoms, orbitals, and projectors:    250   2250   4000

outcell: Unit cell vectors (Ang):
        0.000000    2.715000    2.715000
        2.715000    0.000000    2.715000
        2.715000    2.715000    0.000000

outcell: Cell vector modules (Ang)   :    3.839590    3.839590    3.839590
outcell: Cell angles (23,13,12) (deg):     60.0000     60.0000     60.0000
outcell: Cell volume (Ang**3)        :     40.0258
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=18 nrows=18 sparsity=70.7346 nnzs=22918, refcount: 7>
  <dData1D:(new from dSpData1D) n=22918, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...

Attempting to read DM from file... Failed...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=18 nrows=18 sparsity=70.7346 nnzs=22918, refcount: 8>
  <dData2D:DM n=22918 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      38     249

InitMesh: MESH = 24 x 24 x 24 = 13824
InitMesh: Mesh cutoff (required, used) =   150.000   161.974 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -78.645376
siesta: Eions   =       380.799133
siesta: Ena     =       116.606954
siesta: Ekin    =        79.791623
siesta: Enl     =        28.680178
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =         5.516103
siesta: DUscf   =         0.311920
siesta: DUext   =         0.000000
siesta: Ex      =       -54.334625
siesta: Ec      =       -10.130094
siesta: Exc     =       -64.464719
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -212.172015
siesta: Etot    =      -214.357076
siesta: FreeEng =      -214.357076

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -212.172015     -214.357076     -214.357076  1.185233 -4.526621  0.027229
timer: Routine,Calls,Time,% = IterSCF        1       0.058  12.21
   scf:    2     -214.357119     -214.357097     -214.357098  0.000439 -4.518850  0.020343
   scf:    3     -214.357154     -214.357126     -214.357126  0.001314 -4.495786  0.000200
   scf:    4     -214.357126     -214.357126     -214.357126  0.000006 -4.495804  0.000144

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000062378
max |H_out - H_in|      (eV) :     0.0001438423
SCF cycle converged after 4 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      38     249

siesta: E_KS(eV) =             -214.3571

siesta: E_KS - E_eggbox =      -214.3571

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000000    0.000000   -0.000000
----------------------------------------
   Max    0.000000
   Res    0.000000    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.000000    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):      -35.55      -35.55      -35.55       -0.00       -0.00       -0.00
(Free)E + p*V (eV/cell)     -213.4691
Target enthalpy (eV/cell)     -214.3571

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -78.648404
siesta: Eions   =       380.799133
siesta: Ena     =       116.606954
siesta: Ekin    =        79.783423
siesta: Enl     =        28.696642
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =         5.507645
siesta: DUscf   =         0.310147
siesta: DUext   =         0.000000
siesta: Ex      =       -54.332817
siesta: Ec      =       -10.129985
siesta: Exc     =       -64.462802
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -214.357126
siesta: Etot    =      -214.357126
siesta: FreeEng =      -214.357126

siesta: Final energy (eV):
siesta:  Band Struct. =     -78.648404
siesta:       Kinetic =      79.783423
siesta:       Hartree =      13.096324
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =     -54.332817
siesta:         Corr. =     -10.129985
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =     -64.462802
siesta:  Ion-electron =    -100.576977
siesta:       Ion-ion =    -142.197094
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -214.357126
siesta:         Fermi =      -4.495804

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.022186   -0.000000   -0.000000
siesta:    -0.000000   -0.022186   -0.000000
siesta:    -0.000000   -0.000000   -0.022186

siesta: Cell volume =         40.025752 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00024163          0.00024163  Ry/Bohr**3
siesta:           0.02218575          0.02218575  eV/Ang**3
siesta:          35.54549080         35.54549082  kBar
(Free)E+ p_basis*V_orbitals  =        -213.123590
(Free)Eharris+ p_basis*V_orbitals  =        -213.123590

cite: Please see "sl.psml.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-beta1-29-g209a221d1
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       0.901
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       0.901       0.901   100.00
Setup                  1       0.042       0.042     4.67
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       0.858       0.858    95.28
geom_init              1       0.375       0.375    41.67
state_init             1       0.039       0.039     4.29
hsparse                1       0.001       0.001     0.14
overlap                1       0.032       0.032     3.56
Setup_H0               1       0.287       0.287    31.92
naefs                  2       0.000       0.000     0.02
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.015       0.030     3.32
nlefsm                 2       0.125       0.250    27.75
DHSCF_Init             1       0.077       0.077     8.55
DHSCF1                 1       0.006       0.006     0.70
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.071       0.071     7.84
REMESH                 1       0.009       0.009     0.99
REORD                 20       0.000       0.000     0.03
PHION                  1       0.056       0.056     6.18
COMM_BSC              27       0.000       0.000     0.03
POISON                 7       0.000       0.002     0.18
fft                   14       0.000       0.001     0.11
setup_H                5       0.048       0.238    26.48
DHSCF                  6       0.069       0.416    46.20
DHSCF3                 6       0.048       0.286    31.80
rhoofd                 6       0.026       0.159    17.65
XC                     6       0.001       0.006     0.66
GXC-CellXC             6       0.001       0.006     0.65
gridxc@cellXC          6       0.001       0.006     0.64
vmat                   6       0.020       0.119    13.26
IterSCF                4       0.058       0.231    25.65
compute_dm             4       0.010       0.040     4.49
diagon                 4       0.010       0.040     4.47
c-eigval              56       0.000       0.013     1.42
c-buildHS             56       0.000       0.011     1.26
cdiag                112       0.000       0.004     0.44
cdiag1               112       0.000       0.000     0.04
cdiag2               112       0.000       0.001     0.06
cdiag3               112       0.000       0.003     0.29
c-eigvec              56       0.000       0.014     1.56
cdiag4                56       0.000       0.000     0.01
c-buildD              56       0.000       0.013     1.47
MIXER                  3       0.000       0.000     0.03
PostSCF                1       0.252       0.252    27.94
DHSCF4                 1       0.130       0.130    14.39
dfscf                  1       0.123       0.123    13.67
overfsm                1       0.004       0.004     0.39
writeHSX               1       0.000       0.000     0.03
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.03
optical                1       0.000       0.000     0.00
  

>> End of run:  14-DEC-2023  13:38:21
Job completed
